calicoctl/commands/printer.go

```diff
@@ -207,17 +207,18 @@ func join(items interface{}, separator string) string {
 
 // config returns a function that returns the current global named config
 // value.
-func config(client *client.Client) func(string) string {
+func config(client client.Interface) func(string) string {
 	var asValue string
 	return func(name string) string {
 		switch strings.ToLower(name) {
 		case "asnumber":
 			if asValue == "" {
-				if asn, err := client.Config().GetGlobalASNumber(); err != nil {
-					asValue = "unknown"
-				} else {
-					asValue = asn.String()
-				}
+				//if asn, err := client.Config().GetGlobalASNumber(); err != nil {
```

Comment:
Could you add a TODO here (I know we don't do that -- but this seems like an easy bit of code to overlook when we come to handle the config options)